.rspec

```diff
@@ -3,4 +3,5 @@
 --profile
 --require spec_helper
 --format documentation
+--pattern spec/*_spec*.rb
```

Comment:
it's cargo culted from another project we don't use that convention so will remove.